src/index.ts

```diff
@@ -174,95 +65,15 @@ export default class OtaClient {
     /**
      * List of files in distribution
      */
-    async listFiles(): Promise<string[]> {
-        return (await this.manifest).files;
+    async getContent(): Promise<Manifest['content']> {
```

Comment:
There might be a need to get content for a specific language. Maybe we need one more method - `getLanguageContent`, what do you think?